lib/diagnostic.rb

```diff
@@ -27,7 +39,11 @@
 # which returns the string "think think think".
 
 ##
-# your code here
+class Developer < Person
+  def solve_problems
+    returns "think think think"
```

Comment:
It's `return`, not `returns`. Also, an explicit `return` is not necessary in Ruby, since a Ruby method will default to returning the last evaluated result in its body.